Smash_Combos.Core/Cqrs/Combos/GetCombo/GetComboRequestHandler.cs

```diff
@@ -22,11 +22,24 @@ public GetComboRequestHandler(IDbContext context, IMapper mapper)
 
         public async Task<GetComboResponse> Handle(GetComboRequest request, CancellationToken cancellationToken)
         {
-            var combo = await _dbContext.Combos.Where(combo => combo.Id == request.ComboID).Include(combo => combo.User).Include(combo => combo.Comments).ThenInclude(comment => comment.User).FirstOrDefaultAsync();
+            var combo = await _dbContext.Combos.Where(combo => combo.Id == request.ComboID).Include(combo => combo.User).Include(combo => combo.Comments).FirstOrDefaultAsync();
 
             if (combo == null)
                 return null;
 
+            if(combo.Comments?.Count > 0)
```

Comment:
If you do the same thing with this response object, as the other, then this also gets removed.